CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# Contributing to Garden Linux
+
+## Code of conduct
+
+All members of the Garden Linux community must abide by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Only by respecting each other can we develop a productive, collaborative community. Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting gardenlinux.opensource@sap.com and/or a Garden Linux project maintainer.
+
+## Contributing
+
+Garden Linux uses GitHub to manage reviews of pull requests.
+
+If you are a new contributor see: [Steps to Contribute](#Steps-to-Contribute)
+
+If you have a trivial fix or improvement, go ahead and create a pull request, addressing (with @...) a suitable maintainer of this repository (see CODEOWNERS of the repository you want to contribute to) in the description of the pull request.
```

Comment:
is the maintainer highlight really needed? I think that's put a too high burden on newcomers to figure out which maintainer is appropriate

Reply:
good point, I removed everything following "create a pull request"